### PR TITLE
Nodejs bump

### PR DESCRIPTION
**Pull request by gmokki**
- bump nodejs and npm to latest LTS versions
- remove IE shims
- bump npm packages - and fix one line in gruntfile to keep it working
- bump lodash and momentjs in grunt - keep all other as is to avoid breaking things

### CODE REVIEW

File: nflow-explorer/package.json [hunk 1]
```diff
@@ -4,58 +4,58 @@
...
+    "node-sass": "7.0.1",
     "phantomjs-prebuilt": "2.1.16",
     "protractor": "7.0.0",
     "protractor-http-mock": "0.10.0",
-    "selenium-standalone": "6.17.0",
-    "serve-static": "1.14.1",
-    "sinon": "7.5.0",
+    "selenium-standalone": "8.0.8",
+    "serve-static": "1.14.2",
+    "sinon": "12.0.1",
```
Comment (eputtone) [inline nflow-explorer/package.json:58]: Starting from version 8.0.0, sinon requires that the browser provides `Map`. The used phantomjs does not, so the latest possible sinon version is 7.5.0.
```
12 01 2022 12:39:41.791:INFO [launcher]: Launching browsers PhantomJS with concurrency unlimited
12 01 2022 12:39:41.794:INFO [launcher]: Starting browser PhantomJS
12 01 2022 12:39:41.922:INFO [PhantomJS 2.1.1 (Linux x86_64)]: Connected on socket DYMLqi69YyPh6lr0AAAB with id 51113872
PhantomJS 2.1.1 (Linux x86_64) ERROR
  ReferenceError: Can't find variable: Map
  at node_modules/sinon/pkg/sinon.js:4115:35

PhantomJS 2.1.1 (Linux x86_64): Executed 0 of 0 ERROR (0.083 secs / 0 secs)
```

File: nflow-explorer/src/index.html [hunk 2]
```diff
@@ -60,7 +46,6 @@
     <script src="bower_components/dagre-d3/dist/dagre-d3.core.js"></script>
     <script src="bower_components/chart.js/dist/Chart.js"></script>
     <script src="bower_components/angular-chart.js/dist/angular-chart.js"></script>
-    <script src="bower_components/svg-pan-zoom/dist/svg-pan-zoom.js"></script>
```
Comment (eputtone) [inline nflow-explorer/src/index.html:63]: Why is this removed? Though not superfatal, but removing this line breaks zooming for large workflows, and you always get errors to browser console. Example of large workflow is in nflow-tests (wordGenerator workflow). It's easily started by:
```
java -jar nflow-tests/target/nflow-tests-7.4.1-SNAPSHOT.jar
```

Comment (gmokki) [reply nflow-explorer/src/index.html:63]: should not be, it is just that it is automatically removed from there and I already restored it 2 times. but latest commit again removed it...

**Review by efonsell — APPROVED** [review]
LGTM
